YoYoProject/YoYoProject.csproj

```diff
@@ -39,6 +39,8 @@
     <None Include="App.config" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="Common\GameScale.cs" />
```

Comment:
I'm conflicted about putting these enums into their own files. It's excessive clutter.
Put both enums in GMOptionsBaseModel.cs